com.reprezen.swagedit/src/com/reprezen/swagedit/validation/Validator.java

```diff
@@ -49,8 +50,15 @@
 
 	private static final JsonSchemaManager schemaManager = new JsonSchemaManager();
 
-	private final JsonReferenceValidator referenceValidator = 
-			new JsonReferenceValidator(new JsonReferenceFactory());
+	private final JsonReferenceValidator referenceValidator;
+
+	public Validator(JsonReferenceValidator referenceValidator) {
+		this.referenceValidator = referenceValidator;
+	}
+	
+	public Validator() {
+		this.referenceValidator = new JsonReferenceValidator(new JsonReferenceFactory());
```

Comment:
Why don't we call `this(new JsonReferenceValidator(new JsonReferenceFactory()))` to make it DRYer?